kessler/time_utils.py

```diff
@@ -0,0 +1,81 @@
+import datetime
```

Comment:
@ANDREWNGT I think that the content of this file (`time_utils.py`) can go into `util.py`: could you please move it there?